.editorconfig

```diff
@@ -0,0 +1,10 @@
+# editorconfig.org
```

Comment:
We haven't been including these since they're generally developer- or team-specific. We should remove here and add to .gitignore.

Reply:
k dokey. I just went through CRWA and added changes since this was created...